### PR TITLE
Separate ciscospark plugins into standalone packages

### PR DESCRIPTION
**Pull request by ianwremmel**
- [x] document auth plugins
- [x] document config properties correctly
- [x] fix public/private docs
- [x] fix describe nesting

### CODE REVIEW

File: packages/node_modules/@ciscospark/plugin-phone/src/config.js [hunk 1]
```diff
@@ -2,46 +2,47 @@
  * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
  */
 
+/**
+ * @typedef {Object} PhoneConfig
+ * @property {number} [audioBandwidthLimit=64000]
+ * @property {number} [videoBandwidthLimit=1000000]
+ * @property {boolean} [enableExperimentalGroupCallingSupport=false] - Group
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/plugin-phone/src/config.js]: Should `hangupIfLastActive` be defined here with the HangupBehavior type?

File: packages/node_modules/@ciscospark/plugin-logger/src/config.js [hunk 2]
```diff
@@ -2,18 +2,22 @@
  * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
  */
 
+/**
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/plugin-logger/src/config.js]: I like these defined up here, makes it much easier to read.

File: packages/node_modules/@ciscospark/plugin-logger/src/logger.js [hunk 3]
```diff
@@ -149,6 +156,7 @@ const Logger = SparkPlugin.extend({
    * @instance
    * @memberof Logger
    * @private
+   * @memberof Logger
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/plugin-logger/src/logger.js]: Already have `@memberof` two lines up

File: packages/node_modules/@ciscospark/plugin-logger/src/config.js (hunk 2 above)
Comment (ianwremmel) [reply packages/node_modules/@ciscospark/plugin-logger/src/config.js]: haha, it wasn't really by choice; it was the only way i could get the docs generator to nest things in anyway approaching something reasonable.

File: packages/node_modules/@ciscospark/plugin-phone/src/config.js (hunk 1 above)
Comment (ianwremmel) [reply packages/node_modules/@ciscospark/plugin-phone/src/config.js]: the `@memberof` in `HangupBehavior` takes care of it and renders slightly better. documentation.js still seems to be my favorite doc generator, but i've definitely gotten more familiar with its rough patches than I'd like.

**Pull request merged**